### PR TITLE
Final page scores and restart button

### PR DESCRIPTION
**Pull request by husseinfaqih**

### CODE REVIEW

File: src/pages/lastPage.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { getQuizData } from '../utils/sessionStorage.js';
+
+export function lastPage() {
+  document.body.innerHTML = '';
+  let correctAnswers = 0;
+  let wrongAnswers = 0;
+  let skippedAnswers = 0;
+  for (let finalQuestions of getQuizData().questions) {
```
Comment (mnvacym) [inline src/pages/lastPage.js:8]: Can you rename this as finalQuestion as it should be single

File: src/pages/questionPage.js [hunk 2]
```diff
@@ -70,7 +71,13 @@ export const initQuestionPage = () => {
 };
 
 const nextQuestion = () => {
-  quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
-  setQuizData(quizData);
-  initQuestionPage();
+  //>>>LAST PAGE >>>
+  if (quizData.currentQuestionIndex < 9) {
```
Comment (mnvacym) [inline src/pages/questionPage.js:75]: Instead of hard-coded number can you use `quizData.questions.length - 1`

**Review by mnvacym — CHANGES_REQUESTED** [review]

**Comment by mnvacym** [comment]
Can you also center the restart quiz button on mobile view

image

File: src/pages/lastPage.js (hunk 1 above)
Comment (husseinfaqih) [reply src/pages/lastPage.js:8]: done!

File: src/pages/questionPage.js (hunk 2 above)
Comment (husseinfaqih) [reply src/pages/questionPage.js:75]: done!

**Review by mnvacym — APPROVED** [review]

**Pull request merged**